docs/welcome/introduction.mdx

```diff
@@ -8,7 +8,7 @@ ParadeDB is an Elasticsearch alternative built on Postgres. We're modernizing th
 starting with real-time search and analytics.
 
 ParadeDB is not a fork of Postgres, but regular Postgres with custom extensions installed. ParadeDB itself
-ships with Postgres 16.
+ships with Postgres 16 by default, but we also provide builds for Postgres 13 to 17.
```

Comment:
We should probably link the dockerHub tags page here so users can see how to fetch them.

We should also add a note in the Docker deployment page that we default to 16 but other tags can be pulled. It should be in docs/deploy/docker